InternetHospital.UnitTests/Appointments/AppointmentServiceUnitTests.cs

```diff
@@ -223,6 +223,100 @@ private List<AppointmentForPatient> GetExpectedPatientAppointments(Appointment a
             return new List<AppointmentForPatient> { model };
         }
 
+        [Fact]
+        public void ShouldSubscribeForAppointment()
+        {
+            // arrange
+            const int ALLOWED_STATUS_ID = 1;
+            const int PATIENT_ID = 92;
+            var options = DbContextHelper.GetDbOptions(nameof(ShouldSubscribeForAppointment));
+            var fixture = FixtureHelper.CreateOmitOnRecursionFixture();
+
+            var fixturePatient = fixture.Build<User>()
+                .Create();
+
+            var patientId = fixturePatient.Id;
+
+            var fixtureAppointment = fixture.Build<Appointment>()
+                                           .With(a => a.StatusId, ALLOWED_STATUS_ID)
+                                           .With(a => a.Status, new AppointmentStatus
+                                           {
+                                               Id = ALLOWED_STATUS_ID
+                                           })
+                                           .With(a => a.UserId, PATIENT_ID)
+                                           .With(a => a.User, new User
+                                           {
+                                               Id = PATIENT_ID
```

Comment:
As i understand its even better to remove this value, because you subscribing to this appointment and it already contains User and UserId.